client/gutenberg/extensions/tiled-gallery/constants.js

```diff
@@ -0,0 +1,34 @@
+/** @format */
+
+/**
+ * External Dependencies
+ */
+import { _x } from '@wordpress/i18n';
+
+export const ALLOWED_MEDIA_TYPES = [ 'image' ];
+export const DEFAULT_COLUMNS = 3;
+export const DEFAULT_GALLERY_WIDTH = 580;
+export const DEFAULT_LAYOUT = 'rectangular';
+export const LAYOUT_STYLES = [
+	{
+		label: _x( 'Tiled mosaic', 'Tiled gallery layout', 'jetpack' ),
+		name: 'rectangular',
+		isDefault: true,
+	},
+	{
+		label: _x( 'Tiled columns', 'Tiled gallery layout', 'jetpack' ),
+		name: 'columns',
+	},
+	{
+		label: _x( 'Square tiles', 'Tiled gallery layout', 'jetpack' ),
+		name: 'square',
+	},
+	{
+		label: _x( 'Circles', 'Tiled gallery layout', 'jetpack' ),
+		name: 'circle',
+	},
+];
```

Comment:
These Gutenberg style definitions are used by [`getActiveStyle`](https://github.com/Automattic/wp-calypso/pull/27458/files#diff-aba368b720f7450e330970e2f15499daR103) also at `view.js`, thus making the frontend dependent on `wp.i18n` althought we're not rendering any strings there. That's probably okay for 1st iteration but need to look into this soon.